app/seed/activities_seed.py

```
@@ -0,0 +1,109 @@
+        
+        # 랜덤 activity_type
+        activity_type = random.choice(ACTIVITY_TYPES)
+        
+        # 랜덤 timestamp (최근 3개월)
+        days_ago = random.randint(0, 90)
+        timestamp = now - timedelta(days=days_ago, hours=random.randint(0, 23))
+        
+        # paper_id (activity_type이 "view", "bookmark", "unbookmark"일 때)
+        paper_id = None
+        if activity_type in ["view", "bookmark", "unbookmark"]:
+            paper = random.choice(paper_ids)
```

Ruff (0.14.5)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)